app/metal-controller-manager/internal/api/api.proto

```diff
@@ -4,8 +4,10 @@ package api;
 
 option go_package = "github.com/talos-systems/sidero/app/metal-controller-manager/internal/api";
 
-service Discovery {
+service Agent {
   rpc CreateServer(CreateServerRequest) returns (CreateServerResponse);
+  rpc SetServerAsAvailable(SetServerAsAvailableRequest)
```

Comment:
I _hate_ this RPC name. Any better ideas?

Reply:
`SetServerAvailability`?  I'm not sure I have a strong concept of its scope, as noted above.

Reply:
Should it be "MarkAsWiped"/"MarkAsClean" (as this is what it does?)

Reply:
`MarkAsClean`